terminology.asciidoc

```diff
@@ -32,13 +32,13 @@ In the EV Charging landscape, different market roles can be identified.
 [cols="3s,9",options="header"]
 |===
 |Role  |Description
-|CPO   |Charging Point Operator: Operates a network of Charge Points.
-|eMSP  |e-Mobility Service Provider: gives EV drivers access to charging services.
+|CPO   |Charging Point Operator. Operates a network of Charge Points.
+|eMSP  |e-Mobility Service Provider. Gives EV drivers access to charging services.
 |Hub   |Can connect one or more CPOs to one or more eMSPs.
-|NAP   |National Access Point: Provides a national Database with all (public) Charging Locations, information can be sent and retrieved from the NAP (that makes it different from a typical NSP).
-|NSP   |Navigation Service Provider: Provides EV drivers with location information of Charge Points.
-|Roaming Hub |See: Hub
-|SCSP  |Smart Charging Service Provider: Provided Smart Charging service to other parties might use a lot of different inputs to calculate Smart Charging Profiles.
+|NAP   |National Access Point. Provides a national database with all (public) charging locations. Information can be sent and retrieved from the NAP. This makes it different from a typical NSP.
+|NSP   |Navigation Service Provider. Similar to an eMSP. Provides EV drivers with location information of Charge Points. Usually only interested in Location information.
```

Comment:
I don't like the part: Similar to an eMSP. An NSP only provides location information, no other services. So in my view they are not similar.

Reply:
I took out that line and committed it.  Did it go into the Pull Request or do I need to make another Pull Request every time I need to make changes to my changes?